modules/images-configuration-registry-mirror.adoc

```diff
@@ -66,53 +83,69 @@ In this example, you have a container image registry that is named `example.io`
-<2> Indicates multiple mirror repositories for each target repository. If one mirror is down, the target repository can use another mirror.
-<3> Indicates the registry and repository containing the content that is mirrored.
-<4> You can configure a namespace inside a registry to use any image in that namespace. If you use a registry domain as a source, the `ImageContentSourcePolicy` resource is applied to all repositories from the registry.
-<5> If you configure the registry name, the `ImageContentSourcePolicy` resource is applied to all repositories from a source registry to a mirror registry.
-<6> Pulls the image `mirror.example.net/image@sha256:...`.
-<7> Pulls the image `myimage` in the source registry namespace from the mirror `mirror.example.net/myimage@sha256:...`.
-<8> Pulls the image `registry.example.com/example/myimage` from the mirror registry `mirror.example.net/registry-example-com/example/myimage@sha256:...`. The `ImageContentSourcePolicy` resource is applied to all repositories from a source registry to a mirror registry `mirror.example.net/registry-example-com`.
-
-. Create the new `ImageContentSourcePolicy` object:
+<1> Indicates the API to use with this CR. This must be `config.openshift.io/v1`.
+<2> Indicates the type of image pull CR, either:
+** `ImageDigestMirrorSet`. Pulls a digest-based image.
+** `ImageTagMirrorSet`. Pulls a tag-based image.
+<3> Indicates the type of image pull method, either:
+** `imageDigestMirrors` for an `ImageDigestMirrorSet` CR.
+** `imageTagMirrors` for an ImageTagMirrorSet` CR. 
+<4> Indicates the name of the mirrored image registry and repository.
+<5> Indicates a seondary mirror repository for each target repository. If one mirror is down, the target repository can use another mirror.
+<6> Indicates the registry and repository source, which is the repository that is referred to in image pull specifications.
+<7> Indicates the fallback policy if the image pull fails:
+** `AllowContactingSource` allows continued attempts to pull the image from the repository. This is the default. 
+** `NeverContactSource` prevents continued attempts to pull the image from the repository.
+<8> You can configure a namespace inside a registry to use any image in that namespace. If you use a registry domain as a source, the object is applied to all repositories from the registry.
+<9> If you configure the registry name, the object is applied to all repositories from a source registry to a mirror registry.
+<10> Pulls the image `mirror.example.net/image@sha256:...`.
+<11> Pulls the image `myimage` in the source registry namespace from the mirror `mirror.example.net/myimage@sha256:...`.
+<12> Pulls the image `registry.example.com/example/myimage` from the mirror registry `mirror.example.net/registry-example-com/example/myimage@sha256:...`. The `ImageContentSourcePolicy` resource is applied to all repositories from a source registry to a mirror registry `mirror.example.net/registry-example-com`.
```

Comment:
@QiWang19 Can you PTAL at 10, 11, and 12. They don't seem correct to my reading. But I am not expert here.